DTS-STN/Life-Journeys
# Added GCNotify service to report a problem component

# Description

[LJ-129 Get the "Report a problem" widget to work with Notify](https://lj-decd.atlassian.net/browse/LJ-129?atlOrigin=eyJpIjoiNmMwMTBmMmM5Y2NiNGYzZmE5ZDU1ZjQzY2I1M2I5NmQiLCJwIjoiaiJ9)

Added GCNotify service to Report a Problem. Added the required environment variables to our TeamCity build. Email template was pulled from what Alpha Site has.

## Test Instructions

My docker was acting up (not sure why, potentially webpack issues) so I wasn't able to do the below. Instead I input the values manually into ```report-a-problem.js``` and did ```npm run dev```

1. Pull in branch
2.  In your Dockerfile, specify these values 
```
ENV REPORT_A_PROBLEM_ENABLED=true
ENV NOTIFY_BASE_API_URL=https://api.notification.canada.ca
ENV NOTIFY_API_KEY="ffe6ec79-f1c8-49d4-9917-8b0175e86d50"
ENV NOTIFY_REPORT_A_PROBLEM_EMAIL="" (insert your email here, may have to add to safelist on GCNotify if you're not invited to the team yet)
ENV NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID="9830c589-9522-4fb3-9a6d-603d5546bd8d"
```
3. Do a docker build and run
4. Expand report a problem dropdown, select any number of the checkboxes (must select at least 1), write something in the optional text box (optional, as its name implies, but good to do this to test)
5. Hit submit and you should see the email in your inbox and the record of the email on GCNotify

## Meets Definition of Done
